config/default.yaml

```diff
@@ -120,6 +120,9 @@ user:
 transcoding:
   enabled: true
   threads: 1
+  concurrency: 1
+  niceness: 15 # from 1 to 18
+  ttl: '2 days' # 2 days, transcoding could be long. eg. '2 days' or '12 hours'
```

Comment:
Maybe add a comment to explain what this `ttl` is (so admins understand this parameter is used to kill too long transcoding jobs).